DataFormats/FTLDigi/interface/BTLSample.h

```diff
@@ -35,14 +35,16 @@ class BTLSample {
   void setToA(uint16_t toa)             { setDataWord(toa,  kToAMask,    kToAShift);    }
   void setToA2(uint16_t toa2)           { setDataWord(toa2, kToA2Mask,   kToA2Shift);   }
   void setData(uint16_t data)           { setDataWord(data, kDataMask,   kDataShift);   }
-  void set(bool thr, bool mode, uint16_t toa2, uint16_t toa, uint16_t data) 
+  void set(bool thr, bool mode, uint16_t toa2, uint16_t toa, uint16_t data, uint8_t row, uint8_t col) 
   { 
     flag_  = ( ( (uint16_t)thr  & kThreshMask ) << kThreshShift | 
                ( (uint16_t)mode & kModeMask   ) << kModeShift     );    
 
     value_ = ( ( (uint32_t)toa2 & kToA2Mask   ) << kToA2Shift   | 
                ( (uint32_t)toa  & kToAMask    ) << kToAShift    | 
                ( (uint32_t)data & kDataMask   ) << kDataShift     );    
+    row_ = row;
```

Comment:
why are these stored as separate members in BTLSample but packed in ETLSample?

Reply:
Simple : out of bits in the main data word.
I'll try to unify things as time goes on.